communication/best-practices.md

```diff
@@ -0,0 +1,85 @@
+# Mailing list and Github usage best practice                                                       
+                                                                                
+The Kubernetes Mailing list or Google Groups functions as the primary means of  
+asynchronous communication for the project's                                    
+[Special Interest Groups (SIG)][sig-list] and [Working Groups (WG)][sig-list].  
+That's why you may want to set your filters in your Gmail account to attain a
+good signal-to-noise ratio with regards to the mailing list messages and Github
+notifications.
+
+- [Example for setting filters to Kubernets Mailing lists](#example-for-setting-filters-to-kubernets-mailing-lists)
+- [Examples for setting filters to Kubernetes Github notifications](#examples-for-setting-filters-to-kubernetes-github-notifications)
+                                                                                
+## Example for setting filters to Kubernets Mailing lists           
```

Comment:
IIUC the instructions below in this section show how to create a filter in Gmail, but not specific to Kubernetes mailing lists...

Personally, I use this (if it's helpful):

```
Matches: list:"kubernetes-dev@googlegroups.com"
Do this: Apply label "lists/kubernetes-dev"
```